boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -33,7 +33,7 @@ enabled by moving a 0-ohm resistor.
 Connections and IOs
 ===================
 
-The TTGO T8-C3 board configuration supports the following hardware features:
+The ``ttgo_t8c3`` board target configuration supports the following hardware features:
```

Comment:
Please only rename here, I.e. when directly referring to the Zephyr board target. All the other occurrences of the board name are fine to keep (and actually much more readable). 
Applies to the other file, and there is actually no need to change anything there AFAICS

Reply:
Ok, fixed.